stubs/decorator/decorator.pyi

```diff
@@ -65,8 +64,7 @@ def decorator(
     caller: Callable[..., Any], _func: Callable[..., Any] | None = ...
 ) -> Callable[[Callable[..., Any]], Callable[..., Any]]: ...
 
-class ContextManager(_GeneratorContextManager[_T]):
-    def __call__(self, func: _C) -> _C: ...
```

Comment:
Why did you remove `__call__` here? The `decorator` library explicitly adds it and it's not part of the inheritance tree:

https://github.com/micheles/decorator/blob/519cc713878df4bfb768edfaa65e4bf2d875e421/src/decorator.py#L314-L318

Reply:
Because the stub tests failed complaining that the `__call__` definition here no longer conformed to the parent class declaration of `ContextDecorator.__call__` (it was the same issue as in `jax`: the return type *isn't* the same as the input type, it just has the same call signature).

Deleting the stub in the subclass means it inherits the `ContextDecorator.__call__` signature, which is also correct for `decorator.ContextManager`.

I didn't check if the nominal signature of `decorator.decorate` itself might need updating, though.

Reply:
I edited the PR description to include that additional information about the change fixing a genuine bug in the old signature definition.

Reply:
Whoops, I missed some of the base classes of `_GeneratorContextManager` when checking (twice!).